webapp/components/Editor/Editor.vue

```diff
@@ -70,7 +69,9 @@ export default {
     }
   },
   computed: {
-    ...mapGetters({ placeholder: 'editor/placeholder' }),
+    placeholder() {
+      return this.$t('editor.placeholder')
+    },
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-09-10T13:03:46Z" title="Tuesday, September 10th 2019, 3:03:46 pm +02:00">Sep 10, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -70,7 +69,9 @@ export default {
     }
   },
   computed: {
-    ...mapGetters({ placeholder: 'editor/placeholder' }),
+    placeholder() {
+      return this.$t('editor.placeholder')
+    },
```


much better solution!!
state is the root of all evil!